src/azure-cli/azure/cli/command_modules/cdn/_help.py

```diff
@@ -323,3 +346,360 @@
+"""
+
+helps['cdn waf policy managed-rule-set rule-group-override set'] = """
+type: command
+short-summary: Add a rule group override to a managed rule set on a CDN WAF policy.
+parameters:
+  - name: --rule-override -r
+    short-summary: Override a rule in the rule group
+    long-summary: |
+        rule overrides are specified as key value pairs in the form "KEY=VALUE [KEY=VALUE ...]".
+        Available keys are 'id', 'action', and 'enabled'. 'id' is required. Valid values for
+        'action' are 'Block', 'Redirect', 'Allow', and 'Log', defaulting to 'Block'. Valid values
+        for 'enabled' are 'Enabled' and 'Disabled', defaulting to 'Disabled'.
+examples:
+  - name: Add a rule group override for SQL injections to DefaultRuleSet_1.0 on a CDN WAF policy.
+    text: |
+        az cdn waf policy managed-rule-set rule-group-override set -g group --policy-name policy \\
+          --rule-set-type DefaultRuleSet --rule-set-version 1.0 -n SQLI -r \\
+          id=942440 action=Redirect enabled=Enabled
+  - name: Add multiple rule group overrides to DefaultRuleSet_1.0 on a CDN WAF policy.
+    text: |
+        az cdn waf policy managed-rule-set rule-group-override set -g group --policy-name policy \\
+          --rule-set-type DefaultRuleSet --rule-set-version 1.0 -n SQLI \\
+          -r id=942440 action=Redirect enabled=Enabled \\
+          -r id=942120 -r id=942100
+"""
+
+helps['cdn waf policy managed-rule-set rule-group-override delete'] = """
+type: command
+short-summary: Remove a rule group override from a managed rule set on a CDN WAF policy.
+examples:
+  - name: Remove the rule group override for SQLI from DefaultRuleSet_1.0 on a CDN WAF policy.
+    text: |
+        az cdn waf policy managed-rule-set rule-group-override delete -g group --policy-name policy \\
+          --rule-set-type DefaultRuleSet --rule-set-version 1.0 -n SQLI
+"""
+
+helps['cdn waf policy managed-rule-set rule-group-override list'] = """
+type: command
+short-summary: List rule group overrides of a managed rule on a CDN WAF policy.
+examples:
+  - name: List rule group overrides of a managed rule on a CDN WAF policy.
+    text: >
+        az cdn waf policy managed-rule-set rule-group-override list -g group --policy-name policy \\
+          --rule-set-type DefaultRuleSet --rule-set-version 1.0
+"""
+
+helps['cdn waf policy managed-rule-set rule-group-override show'] = """
+type: command
+short-summary: Show a rule group override of a managed rule on a CDN WAF policy.
+examples:
+  - name: Get the rule group override for rule group SQLI of DefaultRuleSet_1.0 on a CDN WAF policy.
+    text: >
+        az cdn waf policy managed-rule-set rule-group-override show -g group --policy-name policy \\
+          --rule-set-type DefaultRuleSet --rule-set-version 1.0 -n SQLI
+"""
+
+
+helps['cdn waf policy custom-rule'] = """
+type: group
+short-summary: Manage custom rules of a CDN WAF policy.
+"""
+
+helps['cdn waf policy custom-rule set'] = """
```

Comment:
Again, this has create-or-update semantics because we are required to use PUT requests. I saw in the docs that create-or-update semantics should use set, but if this doesn't apply to sub-collections then I'll change it to add.